ebml/EbmlElement.h

```diff
@@ -280,6 +280,21 @@ class EBML_DLL_API EbmlSemanticContext {
 */
 class EBML_DLL_API EbmlElement {
   public:
+    // callback to tell if the element should be written or not
+    // \return true if the element should be written
+    typedef bool (*ShouldWrite)(const EbmlElement &);
```

Comment:
A `using ShouldWrite = bool (*)(const EbmlElement &)` instead of a `typedef`, please. Thanks.

Reply:
Done